stingray/lightcurve.py

```diff
@@ -194,6 +202,22 @@ def __init__(self, time, counts, err=None, input_counts=True,
                                               self.tstart + self.tseg]]))
         check_gtis(self.gti)
 
+    def change_mjdref(self, new_mjdref):
```

Comment:
additional point: It might be worth thinking about not making this a method, but a function instead. In some situations, it might be confusing to have a method that changes the internal state of the object. 

It might be clearer for the user to have a function `change_mjdref` that takes a `Lightcurve` object as input and returns another `Lightcurve` object, but you understand the use cases better than me, so you'll be better able to tell which makes more sense.